instant-xml-macros/src/meta/namespace.rs

```diff
@@ -0,0 +1,374 @@
+pub use base::Namespace;
+mod base {
```

Comment:
I don't think these inline modules are an improvement. Extracting `namespace` out of `meta` makes sense to me, but leaves very little code in `meta` itself... maybe it then makes sense to move the remainder into `lib.rs`?

Reply:
I would personally want to extract those parts into their own module for now, suspect they will disappear once I'm finished. Thus having a mod.rs files that declares modules and exports only.